### ISSUE TITLE
Quick start Error

### ISSUE THREAD

**Issue by wuchen95** [textflint/textflint#8]
Hi, I just installed the textflint using 'pip install textflint' as recommended, under a python 3.7 environment,  CentOS Linux release 7.8.2003.
And I ran the quick start command 'from textflint import Engine'.
The process downloaded a 764M model.zip and a 10.8M wordnet.zip, then a error appeared:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/data/users/wuchen/anaconda3/envs/textflint/lib/python3.7/site-packages/textflint/__init__.py", line 5, in <module>
    from .input_layer import *
  File "/data/users/wuchen/anaconda3/envs/textflint/lib/python3.7/site-packages/textflint/input_layer/__init__.py", line 3, in <module>
    from .dataset import Dataset
  File "/data/users/wuchen/anaconda3/envs/textflint/lib/python3.7/site-packages/textflint/input_layer/dataset/__init__.py", line 1, in <module>
    from .dataset import Dataset, sample_map
  File "/data/users/wuchen/anaconda3/envs/textflint/lib/python3.7/site-packages/textflint/input_layer/dataset/dataset.py", line 23, in <module>
    sample_map = get_sample_map()
  File "/data/users/wuchen/anaconda3/envs/textflint/lib/python3.7/site-packages/textflint/input_layer/dataset/dataset.py", line 20, in get_sample_map
    filter_str='_sample')
  File "/data/users/wuchen/anaconda3/envs/textflint/lib/python3.7/site-packages/textflint/common/utils/load.py", line 34, in task_class_load
    for module in modules:
  File "/data/users/wuchen/anaconda3/envs/textflint/lib/python3.7/site-packages/textflint/common/utils/load.py", line 27, in module_loader
    yield importlib.import_module(module_name)
  File "/data/users/wuchen/anaconda3/envs/textflint/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ModuleNotFoundError: No module named 'textflint.lib'

**Comment by anaivebird** [comment]
On Google Colab, it works with Python 3.7.10.

Click following link: 
https://colab.research.google.com/drive/1gPgZ8Y6wHqDQ5R9nUoSrEE0WW6uTObQL?usp=sharing

**Comment by anaivebird** [comment]
Duplicate of #6

**Comment by haonan-li** [comment]
The same error exist on my machine, my python version is 3.8.

**Comment by BeyonderXX** [comment]
Hi,
@wuchen95 @haonan-li 
Can you tell me what version of your `importlib` and `pkgutil` packages are?

**Comment by haonan-li** [comment]
Hi,

To reproduce the error, the environment is:

conda=4.9.2
python=3.7.10

After successful installed textflint, 

> $ pip show importlib
> WARNING: Package(s) not found: importlib
> 
> $ pip show pkgutil
> WARNING: Package(s) not found: pkgutil
> 
> $ pip show importlib_metadata
> Name: importlib-metadata
> Version: 3.10.0
> Summary: Read metadata from Python packages
> Home-page: https://github.com/python/importlib_metadata
> Author: Jason R. Coombs
> Author-email: jaraco@jaraco.com
> License: UNKNOWN
> Location: /home/{xxxxxx}/.conda/envs/textflint/lib/python3.7/site-packages
> Requires: zipp, typing-extensions
> Required-by: transformers, spacy, konoha, huggingface-hub, fsspec, datasets, catalogue

**Comment by wuchen95** [comment]
Hi, my environment is almost the same with @haonan-li except that my conda version is 4.8.5

**Comment by wuchen95** [comment]
Hi, my environment is almost the same with @haonan-li except that my conda version is 4.8.5

**Comment by anaivebird** [comment]
@haonan-li @wuchen95 
Could you bother to check whether using latest anaconda with your OS works for TextFlint?

```
wget https://repo.anaconda.com/archive/Anaconda3-2020.11-Linux-x86_64.sh
bash Anaconda3-2020.02-Linux-x86_64.sh
conda init
conda activate base
pip install textflint
python
import textflint
```

Latest anaconda with textflint works on both CentOs and ubuntu on my machines.

**Comment by haonan-li** [comment]
It works with the latest Conda version.

**Comment by anaivebird** [comment]
So I guess something broken in your conda/python env. Detailed probing of actual problem needs providing the minimal steps to get the broken env to reproduce the error you reported. Possible way is finding an old anaconda installation(what you used before) to reproduce the importlib error.

**Comment by wuchen95** [comment]
Hi, the solution worked for me with the latest anaconda.
